src/components/Editor/EditPublications.vue

```diff
@@ -1,4 +1,4 @@
-<template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
+<template xmlns:v-sl ot="http://www.w3.org/1999/XSL/Transform">
```

Comment:
it should be a v-slot. please return it back to what it was.

Reply:
This typo has been fixed.